src/app/common/services/messages/messages.service.ts

```diff
@@ -79,10 +72,10 @@ export class MessagesService {
     const temp8 = 28 + 10 * Math.random();
 
     const message = `[{"bt": 15020, "bn":"temperature", "t": 0, "v":${temp1}},
-                      {"t":10, "v":${temp2}}, {"t":20, "v":${temp3}},
-                      {"t":30, "v":${temp4}}, {"t":40, "v":${temp5}},
-                      {"t":50, "v":${temp6}}, {"t":60, "v":${temp7}},
-                      {"t":70, "v":${temp8}}]`;
+{"t":10, "v":${temp2}}, {"t":20, "v":${temp3}},
+{"t":30, "v":${temp4}}, {"t":40, "v":${temp5}},
+{"t":50, "v":${temp6}}, {"t":60, "v":${temp7}},
+{"t":70, "v":${temp8}}]`;
```

Comment:
I prefer previous tabulation. This one is wrong, you should have a tab between first line and others